engine/base/kafka/src/main/scala/pl/touk/nussknacker/engine/baseengine/kafka/NuKafkaEngineApp.scala

```diff
@@ -28,9 +28,14 @@ object NuKafkaEngineApp extends App with LazyLogging {
 
   val scenarioInterpreter = prepareScenarioInterpreter
 
-  Using.resource(scenarioInterpreter) { interpreter =>
-    interpreter.run()
-  }
+  scenarioInterpreter.run()
+  
+  Runtime.getRuntime.addShutdownHook(new Thread() {
+    override def run(): Unit = {
+      logger.info("Closing interpreter")
```

Comment:
Closing nu ? closing engine ? imo interpreter does not mean anything significant for administrator

Reply:
changed